addons/bestja_detailed_reports/views/project.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="bestja_project_form">
+            <field name="model">bestja.project</field>
+            <field name="inherit_id" ref="bestja_requests.bestja_project_form"/>
```

Comment:
Your module is not dependent on `bestja_requests` (and it shouldn't be), so you can't reference elements defined in `bestja_requests`! This should inherit from `bestja_project_hierarchy.bestja_project_form` instead.